tests/test_ui.py

```diff
@@ -170,6 +170,23 @@ def addSample(self):
         QTest.mouseClick(widget.create_primitive_button, Qt.LeftButton)
         self.assertIsNotNone(self.model.sample)
 
+        self.window.sample_properties_dialog_action.trigger()
+        sample_properties_widget = self.getDockedWidget(self.window.docks, SampleProperties.dock_flag)
+        self.assertTrue(isinstance(sample_properties_widget, SampleProperties))
+        bytes_used = self.model.sample.vertices.nbytes
```

Comment:
unnecessary variable `self.model.sample.vertices.nbytes` is clear

Reply:
bytes_used removed from test_widget.py and test_ui.py